src/components/interactive-map.js

```diff
@@ -282,7 +282,10 @@ export default class InteractiveMap extends PureComponent {
       }
 
       if (this.props.onHover) {
-        const viewport = new WebMercatorViewport(this.props);
+        const viewport = new WebMercatorViewport(Object.assign({}, this.props, {
+          width: this._width,
```

Comment:
This will always override width/height with the internal values?

Reply:
Yes. Internal values will always reflect the numeric dimension of the map.